### PR TITLE
First draft of docs for streaming connector from Braze Currents

### PR DESCRIPTION
**Pull request by jeffreystutz**

### CODE REVIEW

File: help/sources/tutorials/ui/create/marketing-automation/braze.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+solution: Experience Platform
+title: Create a Braze Source Connection in the UI
+type: Tutorial
+description: Learn how to create a Braze source connection using the Adobe Experience Platform UI and Braze Currents
+exl-id: 452b7290-b9e8-4728-8b58-0e0c76bd9449
+---
+# Create a [!DNL Braze] source connection in the UI
+
+Source connectors in Adobe Experience Platform provide the ability to ingest externally sourced data on a scheduled basis. This tutorial provides steps for creating a streaming [!DNL Braze] source connector using the [!DNL Platform] user interface.
+
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/marketing-automation/braze.md:17]: Can we add one or two lines here to talk about what Braze is and maybe a use case example on what customers can use it for? Something like: "Braze is... Use the Braze source to bring {what kinds of data?} from your Braze account to Adobe Experience Platform."

File: help/sources/tutorials/ui/create/marketing-automation/braze.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+*   [[!DNL Experience Data Model (XDM)] System](../../../../../xdm/home.md): The standardized framework by which [!DNL Experience Platform] organizes customer experience data.
+    *   [Basics of schema composition](../../../../../xdm/schema/composition.md): Learn about the basic building blocks of XDM schemas, including key principles and best practices in schema composition.
+    *   [Schema Editor tutorial](../../../../../xdm/tutorials/create-schema-ui.md): Learn how to create custom schemas using the Schema Editor UI.
+*   [[!DNL Real-Time Customer Profile]](../../../../../profile/home.md): Provides a unified, real-time consumer profile based on aggregated data from multiple sources.
+
+This tutorial also requires a working understanding of [[!Braze Currents]https://www.braze.com/docs/user_guide/data_and_analytics/braze_currents).
+
+If you already have a [!DNL Braze] connection, you may skip the remainder of this document and proceed to the tutorial on [configuring a dataflow](../../dataflow/marketing-automation.md).
+
+## Create your [!DNL Platform] Data Source
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/marketing-automation/braze.md]: Let's change this to "Connect your [!DNL Braze] account to Experience Platform

File: help/sources/tutorials/ui/create/marketing-automation/braze.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+Log in to [Adobe Experience Platform](https://platform.adobe.com) and then select **[!UICONTROL Sources]** from the left navigation bar to access the **[!UICONTROL Sources]** workspace. The **[!UICONTROL Catalog]** screen displays a variety of sources from which you can select.
+
+Select the Marketing Automation category from the catalog on the left-hand side of your screen. Alternatively, you can find the specific source you wish to work with using the search option.
+
+Under the **[!UICONTROL Marketing automation]** category, select **[!UICONTROL Braze]**. If this is your first time using this connector, select **[!UICONTROL Configure]**. Otherwise, select **[!UICONTROL Add data]** to create a new [!DNL Braze] connector.
+
+![catalog](../../../../images/tutorials/create/braze/catalog.png)
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/marketing-automation/braze.md]: Let's update the alt-text here from "catalog" to "The sources catalog on the Experience Platform UI with the Braze Currents source selected."

File: help/sources/tutorials/ui/create/marketing-automation/braze.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+The **[!UICONTROL Connect to Braze]** page appears. Upload the provided Braze Currents sample file. This sample file contains all possible fields that Braze might send as part of an event.  Proceed to create a new data set. Map your data set into your schema. Note that Braze timestamps are in seconds, and that you'll need to create calculated fields in milliseconds to properly refelect the timestamps in {!DNL Platform].
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/marketing-automation/braze.md]: We can add screenshots here for the steps: Select Data, Dataflow Detail, and Mapping. But as an alternative we can break this down into steps as well.

Next, you must:
- Upload the provided [!DNL Braze Currents] sample file... this file contains all possible fields that Braze might send as part of an event.(can we add more detail on this provided sample file? From who/where does a customer get the file?)
- "Provide dataflow details, including information on your dataset and the schema that you are mapping to."
- Configure mapping for your data. **Note**: Braze timestamps are measured in seconds and not milliseconds. Therefore, you must create calculated fields in milliseconds to properly reflect the timestamps in Experience Platform (please add an example of the calculated field formula as well, I think it is something like "time * 1000"

File: help/sources/tutorials/ui/create/marketing-automation/braze.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+### Gather required credentials
+
+Once your data source is created in [!DNL Platform] collect the following information, which you'll use to configure [!DNL Braze Currents] to send data into [!DNL Platform]
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/marketing-automation/braze.md]: Let's rephrase this to: "Once your connection is created, you must collect the following credential values, which you will then provide in the Braze UI dashboard (add hyperlink to Braze UI) to send data to Experience Platform.

File: help/sources/tutorials/ui/create/marketing-automation/braze.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+| `Client ID` | The client ID associated with your [!DNL Platform] source. |
+| `Client Secret` | The client secret associated with your [!DNL Platform] source. |
+| `Tenant ID` | The tenant ID associated with your [!DNL Platform] source. |
+| `Sandbox Name` | The sandbox associated with your [!DNL Platform] source. |
+| `Dataflow ID` | The dataflow ID associated with your [!DNL Platform] source. |
+| `Streaming Endpoint` | The streaming endpoint associated with your [!DNL Platform] source. |
+
+### Configure [!DNL Braze Currents] to stream data to your data source
+
+Within the [!DNL Braze Dasshboard], navigate to Partner Integrations -> Data Export, then click "Create New Current". You will be prompted to provide a name for the connector, contact information for notifications about the connector, and the credentials listed above. Select the events you wish to receive and optionally configure any field exclusions/transformations and click "Launch Current".
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/marketing-automation/braze.md]: - typo with "Dasshboard"
- We should add a note or important tag here that when a customer retrieves their streaming endpoint URL, they must manually use the BATCH endpoint URL instead, they have to add "/batch" in the URL that is provided on Experience Platform

**Review by carlojavier — CHANGES_REQUESTED** [review]
@jeffreystutz Thanks for making the PR. I've added a few comments for you. Are there any other prerequisites that customers have to know before they can use Braze on Experience Platform? Just want to make sure we have all that covered. Thanks! Let me know if you have any questions

File: help/sources/tutorials/ui/create/marketing-automation/braze.md (hunk 1 above)
Comment (jeffreystutz) [reply help/sources/tutorials/ui/create/marketing-automation/braze.md:17]: Great feedback, will add that in.

File: help/sources/tutorials/ui/create/marketing-automation/braze.md (hunk 4 above)
Comment (jeffreystutz) [reply help/sources/tutorials/ui/create/marketing-automation/braze.md]: Breaking down into steps is a great idea. Is the sample file typically hosted in AEP docs or partner (Braze)? I'm fine either way, just let me know.

And yeah screenshots are a good idea. It wasn't clear to me if it was expected that I would upload the AEP screenshots or if that was something Adobe would handle. Can you please advise?

File: help/sources/tutorials/ui/create/marketing-automation/braze.md (hunk 5 above)
Comment (jeffreystutz) [reply help/sources/tutorials/ui/create/marketing-automation/braze.md]: Note that there isn't a one-size-fits-all link into the exact spot into the UI, but I'll get as close as I can.

File: help/sources/tutorials/ui/create/marketing-automation/braze.md (hunk 6 above)
Comment (jeffreystutz) [reply help/sources/tutorials/ui/create/marketing-automation/braze.md]: Our UI autocorrects to the batch endpoint. Should we still add the note? If the interface will remain static (ie we can be confident that adding /batch is the right thing to do for the foreseeable future), it seems like a better UX to not mention it.

Noted on the typo.

File: help/sources/tutorials/ui/create/marketing-automation/braze.md (hunk 4 above)
Comment (carlojavier) [reply help/sources/tutorials/ui/create/marketing-automation/braze.md]: - Re: sample file: I think we can make the call on this one. I think it would be good to be hosted in both sides. What do you think?
- Re: screenshots: Partner generally uploads the screeenshots since I may sometimes not have access to the source being integrated. I can make updates for maintenance though, once they are live. We only only one screenshot for each of the three steps. Please use this hex code too if possible: 19E641. Thanks!

File: help/sources/tutorials/ui/create/marketing-automation/braze.md (hunk 6 above)
Comment (carlojavier) [reply help/sources/tutorials/ui/create/marketing-automation/braze.md]: Ahh that is great then. I think we should note then that the Braze UI will auto-correct the endpoint.

File: help/sources/tutorials/ui/create/marketing-automation/braze.md (hunk 4 above)
Comment (jeffreystutz) [reply help/sources/tutorials/ui/create/marketing-automation/braze.md]: Where/how would you like me to use that hex code?

Comment (carlojavier) [reply help/sources/tutorials/ui/create/marketing-automation/braze.md]: For the markups on screenshots. Ideally, we have the "actions" marked up. Similar to: https://experienceleague.adobe.com/docs/experience-platform/sources/ui-tutorials/on-demand-ingestion.html?lang=en 

I can also mark up the screenshots once I port this PR over to the internal repo. I just don't have access to an org with Braze Currents source yet and can't get the screenshots until after we release

Comment (jeffreystutz) [reply help/sources/tutorials/ui/create/marketing-automation/braze.md]: If it's all the same to you on the sample file, I'd like to host it on our public examples repo. That'll make it easier to maintain.

Comment (carlojavier) [reply help/sources/tutorials/ui/create/marketing-automation/braze.md]: okay that sounds good to me!

**Comment by jeffreystutz** [comment]
Believe I addressed all your feedback, please let me know if you have anything further.

**Comment by carlojavier** [comment]
> Believe I addressed all your feedback, please let me know if you have anything further.

Looks good. I will bring this over to our internal repo

**Review by carlojavier — APPROVED** [review]

**Comment by carlojavier** [comment]
@jeffreystutz closing this PR. We've merged and published a copy on the internal repo here: https://experienceleague.adobe.com/docs/experience-platform/sources/ui-tutorials/create/marketing-automation/braze.html?lang=en Thanks for working on this!
